cmake/toolchain/support/arduino-support.cmake

```diff
@@ -17,7 +17,10 @@ set(FPRIME_ARDUINO TRUE)
 set(ARDUINO_WRAPPER_JSON_OUTPUT "${CMAKE_BINARY_DIR}/arduino-cli-compiler-settings.json")
 set(EXTRA_LIBRARY_SOURCE "${CMAKE_CURRENT_LIST_DIR}/sources/extras.cpp")
 
-
+# Set the executable suffix if not defined
+if (NOT DEFINED FPRIME_ARDUINO_EXECUTABLE_SUFFIX)
+    set(FPRIME_ARDUINO_EXECUTABLE_SUFFIX ".elf")
+endif()
```

Comment:
Not sure if this was intended but every build now creates a `<deployment>.elf.x` file, where `x` can be `.bin`, `.hex`, etc.

So for Teensy builds, the file I upload to my board ends up being named `LedBlinker.elf.hex` instead of `LedBlinker.hex`. The `.elf` extension does not do anything to the binary, I can still upload and run the file on my teensy, but I just got confused with the sudden `.elf` in the middle.

Likewise for my ESP32 build I get `LedBlinker.elf.bin` instead of just `LedBlinker.bin`.

Reply:
This is because the files are just named after the previous (input) file.  I say this is ok behavior, but also I am biased (less work).